### DISCUSSION TITLE
Problem in DiamondVue[5.X.X]

### DISCUSSION THREAD

**Discussion by Rekl0w** [primefaces/community#1541] [PrimeVue Templates]
![Screenshot 1](https://github.com/primefaces/community/assets/91488737/20a8d5d1-3904-4813-99a7-ef387d4b0852)
![Screenshot 2](https://github.com/primefaces/community/assets/91488737/e92867b5-7d4c-4b6b-8afc-b4f64bc752d3)
![Screenshot 3](https://github.com/primefaces/community/assets/91488737/85e01572-72e8-4c02-9cd5-6a968d881dc0)

Hi,

There is an issue when we trying to toggle the tiered menu at the datatable's first data, when we click the first data's tiered menu button, the menu ref is null, but rest normally toggles. First screenshot is releated the normal toggle, second one is when we search a data at the datatable and where we get the error, third one is the console log of the second screenshot's data's tiered menu button toggle.

There is the releated code.

```

const menu = ref();

const toggleFunc = async (event, data) => {
    menu.value.toggle(event); -> this is the error's location, it says menu.value is null
};

<Column header="asd" style="min-width: 7rem" frozen alignFrozen="right">
                <template #body="{ data }">
                    <div class="flex justify-content-center">
                        <Button type="button" rounded icon="pi pi-ellipsis-v" @click="toggleFunc($event, data)" aria-haspopup="true" aria-controls="overlay_tmenu" />
                        <TieredMenu ref="menu" id="overlay_tmenu" :model="items" popup />
                    </div>
                </template>
            </Column>
```

Thank you.

**Comment by evrenonur** [comment]
@tugcekucukoglu I just noticed the same problem

**Comment by tugcekucukoglu** [comment]
In which demo did you encounter this problem?

**Comment by Rekl0w** [comment]
I am currently using 5.1.0 but i tried this at 5.3.0 too.

**Comment by Rekl0w** [comment]
Any updates here ? @tugcekucukoglu
